admin/controllers/results.js

```diff
@@ -36,10 +36,12 @@ controller.getViewResultsPage = async (req, res, next) => {
     return next(error)
   }
   const currentDate = moment.tz(req.user.timezone || config.DEFAULT_TIMEZONE)
-  const isResultsPageAccessible = schoolHomeFeatureEligibilityPresenter.isResultsPageAccessible(currentDate, checkWindow)
-  const nationalScore = resultPresenter.getScoreWithOneDecimalPlace(checkWindow.score)
-  schoolScore = resultPresenter.getScoreWithOneDecimalPlace(schoolScoreRecord && schoolScoreRecord.score)
-  if (!isHdfSubmitted || !isResultsPageAccessible) {
+
+  const isResultsPageAccessibleForSubmittedHdfs =
```

Comment:
maybe i don't understand the context well enough, but it feels like you are appending some new logic to this method, outside the actual method. 🤔   or, is this just a single situation where the logic is different to other uses?

Reply:
@GuyHarwood all the logic lies within the presenter now.